docs/troubleshooting/clusterResourcePlacementAvailable.md

```diff
@@ -0,0 +1,205 @@
+    version: v1
+  - group: apps
+    kind: Deployment
+    name: my-deployment
+    namespace: test-ns
+    version: v1
+ ```
+In the `ClusterResourcePlacement` status, within the `failedPlacements` section for `kind-cluster-1`, we get a clear message
+as to why the resource failed to apply on the member cluster. Immediately preceding this in the conditions section,
+the `Available` condition for `kind-cluster-1` is flagged as false, citing the `NotAllWorkAreAvailable` reason.
+This signifies that the Work object intended for the member cluster `kind-cluster-1` is not yet available.
+
+To gain more insights also take a look at the `work` object, please check this [section](#how-and-where-to-find-the-correct-work-resource) for more details,
+
+### Work status of kind-cluster-1:
+```
+status:
+conditions:
+- lastTransitionTime: "2024-05-14T18:52:31Z"
+  message: Work is applied successfully
+  observedGeneration: 1
+  reason: WorkAppliedCompleted
+  status: "True"
+  type: Applied
+- lastTransitionTime: "2024-05-14T18:52:31Z"
+  message: Manifest {Ordinal:1 Group:apps Version:v1 Kind:Deployment Resource:deployments
+  Namespace:test-ns Name:my-deployment} is not available yet
+  observedGeneration: 1
+  reason: WorkNotAvailableYet
+  status: "False"
+  type: Available
+  manifestConditions:
+- conditions:
+  - lastTransitionTime: "2024-05-14T18:52:31Z"
+    message: Manifest is already up to date
+    reason: ManifestAlreadyUpToDate
+    status: "True"
+    type: Applied
+  - lastTransitionTime: "2024-05-14T18:52:31Z"
+    message: Manifest is trackable and available now
+    reason: ManifestAvailable
+    status: "True"
+    type: Available
+    identifier:
+    kind: Namespace
+    name: test-ns
+    ordinal: 0
+    resource: namespaces
+    version: v1
+- conditions:
+  - lastTransitionTime: "2024-05-14T18:52:31Z"
+    message: Manifest is already up to date
+    observedGeneration: 1
+    reason: ManifestAlreadyUpToDate
+    status: "True"
+    type: Applied
+  - lastTransitionTime: "2024-05-14T18:52:31Z"
+    message: Manifest is trackable but not available yet
+    observedGeneration: 1
+    reason: ManifestNotAvailableYet
+    status: "False"
+    type: Available
+    identifier:
+    group: apps
+    kind: Deployment
+    name: my-deployment
+    namespace: test-ns
+    ordinal: 1
+    resource: deployments
+    version: v1
+```
+Looking at the status `Available` condition for `kind-cluster-1`, we see that the deployment `my-deployment` is not available yet on the member cluster.
+Therefore, there might be something wrong with the deployment manifest.
+
+#### Resolution:
+In this scenario, a viable solution is to rectify the deployment manifest by verifying that all fields are accurately specified.
+After fixing the resource manifest and updating it, it's essential to delete the CRP and then reapply or recreate it. This step ensures that the changes made to the resource manifest are properly reflected in the CRP configuration.

```

Comment:
Why do we have to delete the CRP here, the new image name changes will be automatically propagated by the existing CRP